### PR TITLE
Swap edit and Select actions on M2O

### LINKED ISSUE #19708: Edit behaviour inconsistent on Many to One field
### Describe the Improvement

When clicking a field, the user expects to be editing the contents of that field. With all other linked collections, this is the case - you click the field and out slides the edit box, with Many To One, it instead slides out the "Select Item" menu (unless you have select disabled). 

While it makes sense from a backend point of view, if select is enabled, technically clicking it and selecting a different item is editing, from an end users perspective this is confusing. 

We've found users are left confused after clicking the large field and a menu with a huge list of items opening instead of being shown the content to edit, them having to exit this and click the much smaller edit target.

chrome_8apwPDn7N7

Proposal:
Swap these two actions, Clicking the larger box should open the edit menu, and have a smaller button to swap the item. Editing is a much more common action than swapping.

### PR DESCRIPTION
**Pull request by D3VL-Jack**
Closes #19708 

chrome_Cxxt1F9rra

The edit and swap behaviour have been swapped such that clicking the larger box opens the edit menu, and the swap moved to a smaller icon at the end.

### CODE REVIEW

File: app/src/lang/translations/en-US.yaml [hunk 1]
```diff
@@ -589,6 +589,7 @@ bookmark_doesnt_exist: Bookmark Doesn't Exist
 bookmark_doesnt_exist_copy: The bookmark you're trying to open couldn't be found.
 bookmark_doesnt_exist_cta: Return to collection
 select_an_item: Select an item...
+change_item: Change Item
```
Comment (paescuj) [inline app/src/lang/translations/en-US.yaml:592]: Might also be "Swap Item" or similar...

**Review by paescuj — DISMISSED** [review]
Thanks for your contribution ❤️ 
Makes sense to me 👍

**Comment by D3VL-Jack** [comment]
Thanks @paescuj for the feedback and tweaks! 

After a discussion in Discord, this PR needs some more work focused around the empty state.
Thread -> https://discord.com/channels/725371605378924594/1152418116127707228

**Comment by paescuj** [comment]
Thanks @D3VL-Jack for this pull request!
I'm closing it for now in favor of a new pull request which will implement it as per https://discord.com/channels/725371605378924594/1152418116127707228.

**Pull request closed without merging**